pkg/kamailio/obs/kamailio.service

```diff
@@ -7,9 +7,13 @@ After=network-online.target
 Type=simple
 User=kamailio
 Group=daemon
+PermissionsStartOnly=true
 Environment='CFGFILE=/etc/kamailio/kamailio.cfg'
 Environment='SHM_MEMORY=64'
 Environment='PKG_MEMORY=4'
+RuntimeDirectory=kamailio
+RuntimeDirectoryMode=0755
+PIDFile=/var/run/kamailio/kamailio.pid
```

Comment:
`PIDFile` is optional put may be accepted.
Could you also describe how `RuntimeDirectory` and `RuntimeDirectoryMode`  help you.

Reply:
Also look [`https://www.freedesktop.org/software/systemd/man/systemd.service.html`](https://www.freedesktop.org/software/systemd/man/systemd.service.html).
```
If set to forking, it is expected that the process configured with ExecStart= will call fork() as part of its start-up. The parent process is expected to exit when start-up is complete and all communication channels are set up. The child continues to run as the main daemon process. This is the behavior of traditional UNIX daemons. If this setting is used, it is recommended to also use the PIDFile= option, so that systemd can identify the main process of the daemon. systemd will proceed with starting follow-up units as soon as the parent process exits.
```
`PIDFile` is used in case process started as working process. Now Kamailio is starts as `simple` process with `-DD` options. Main process is still alive and not need to use `PIDFile`.